### PR TITLE
Add a fat footer :ok_hand:

### PR DESCRIPTION
**Pull request by dtaylor-us**

### CODE REVIEW

**Comment by outcastgeek** [comment]
What's with the lipstick @dtaylor-us ?

File: src/main/ui/templates/user.cljs [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 (defn login-ui [csrf-token]
   [:div.content-container
-   [:form.spacer-top-50 {:action "/user/authenticate", :method "post"}
```
Comment (outcastgeek) [inline src/main/ui/templates/user.cljs]: May use `{:action (path-for :authenticate...`

File: src/main/ui/templates/user.cljs [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
+   [:form.mt5.mb6 {:action "/user/authenticate", :method "post"}
```
Comment (outcastgeek) [inline src/main/ui/templates/user.cljs]: May use `{:action (path-for :authenticate...`

File: src/main/ui/templates/user.cljs [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 (defn register-ui [csrf-token]
   [:div.content-container
-   [:form.spacer-top-50 {:action "/user/enroll", :method "post"}
+   [:form.mt5.mb6 {:action "/user/enroll", :method "post"}
```
Comment (outcastgeek) [inline src/main/ui/templates/user.cljs]: May use `{:action (path-for :enroll...`

**Comment by dtaylor-us** [comment]
image

**Comment by outcastgeek** [comment]
Thank you!

**Pull request merged**